Cargo.toml

```diff
@@ -48,6 +49,11 @@ version = "1"
 [dependencies.thiserror]
```

Comment:
Can we remove hashbrown, since we use ahash? Or vice verca?
Because hashbrown uses ahash.

Reply:
hashbrown is build with ahash, but I don`t think it makes ahash publicly available through its API.
How we use `hashbrown::raw` it require us to set the hash algorithm manually requiring us add `ahash` as a dependency.